metrics/web/test/common/presentation/widgets/metrics_page_title_test.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/common/presentation/widgets/metrics_page_title.dart';
+import 'package:network_image_mock/network_image_mock.dart';
+
+void main() {
+  group("MetricsPageTitle", () {
+    testWidgets(
```

Comment:
Should we add more tests about navigating back? Like shows the back button, navigates back if possible, etc...